packages/teams-js/src/public/meeting.ts

```diff
@@ -414,4 +418,14 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.speakingStateChanged', handler);
   }
+
+  export function registerRaiseHandStateChangedHandler(
+    handler: (raiseHandState: IRaiseHandState | SdkError | null) => void,
+  ): void {
+    if (!handler) {
+      throw new Error('[registerRaiseHandStateChangedHandler] Handler cannot be null');
```

Comment:
Curious: why does this file sometimes throw Error objects and sometimes return SdkError? Seems like the meeting file should be using SdkError for everything

Reply:
The meeting file will end up using SdkError (error code from the SDK) instead of the current JsSdkErrorCode (and error objec) I am speaking with my team and we will coordinate a cleanup plan. That work will be done in a separate PR.